src/app/app.module.ts

```diff
@@ -7,6 +7,21 @@ import { HomePage } from '../pages/home/home';
 import { TabsPage } from '../pages/tabs/tabs';
 import { LoginPage } from '../pages/login/login';
 import { RegisterPage } from '../pages/register/register';
+import { BrowsePage } from '../pages/browse/browse';
+
+
+
+// Import the AF2 Module
+import { AngularFireModule } from 'angularfire2';
+
+// AF2 Settings
+export const firebaseConfig = {
+  apiKey: "AIzaSyCBAaf-Umc90JrkCcublrJniE4QI4XTLD0",
```

Comment:
For now, we have the repo in public and stuff, do we really want those on github and public accessible ? 
Not sure about that How we can set and use those differently, any idea ?